redaxo/src/core/pages/profile.php

```diff
@@ -87,12 +89,16 @@
 }
 
 if (rex_post('upd_psw_button', 'bool')) {
+    $passwordPolicy = rex_backend_password_policy::factory(rex::getProperty('password_policy', []));
```

Comment:
müssen wir wirklich an allen call-sites wissen dass die policy aus dem rex-property kommt oder können wir das in der factory handeln (oder in einer anderen separaten factory methode?)